Formula/trojan-go.rb

```diff
@@ -0,0 +1,114 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go.git",
+      tag:      "v0.10.0",
+      revision: "a9ce6d4ada9143d58e8d1373d23f9ebfe871119e"
```

Comment:
Can we use a tarball instead of a Git checkout?

Reply:
`trojan-go` need commit id to show in its version.
```text
$ trojan-go -version
Trojan-Go v0.10.0
Go Version: go1.16.3
OS/Arch: darwin/amd64
Git Commit: a9ce6d4ada9143d58e8d1373d23f9ebfe871119e
```

hmm.. I will try to do that.

Reply:
I see. A Git checkout is fine then.